src/clean-core/from_string.cc

```diff
@@ -14,6 +14,24 @@ bool parse(cc::string_view s, T& v)
     ss >> v;
     return bool(ss) && ss.eof();
 }
+
+template <>
```

Comment:
function specialization is extremely uncommon and also surprising. just a plain overload suffices here.

Reply:
Fixed